pkgs/development/python-modules/torch/default.nix

```diff
@@ -52,15 +52,15 @@
 
 let
   inherit (lib) lists strings trivial;
-  inherit (cudaPackages) cudatoolkit cudaFlags cudnn nccl;
+  inherit (cudaPackages) cudaFlags cudnn nccl;
 in
 
 assert cudaSupport -> stdenv.isLinux;
 assert cudaSupport -> (cudaPackages.cudaMajorVersion == "11");
 
 # confirm that cudatoolkits are sync'd across dependencies
-assert !(MPISupport && cudaSupport) || mpi.cudatoolkit == cudatoolkit;
-assert !cudaSupport || magma.cudaPackages.cudatoolkit == cudatoolkit;
+assert !(MPISupport && cudaSupport) || mpi.cudatoolkit == cudaPackages.cudatoolkit;
+assert !cudaSupport || magma.cudaPackages.cudatoolkit == cudaPackages.cudatoolkit;
```

Comment:
I guess we're dropping this only later when we remove `cudatoolkit` from `magma`&c?

Reply:
Good catch! MPI hasn't been moved over, but `magma` has so I'll change that one.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/commit/ea939b9d1db4dc72bd24f3459e856ec02238f8ff